server.js

```diff
@@ -22,6 +23,7 @@ app.use(express.json());
 // mount imported routes
 app.use('/api/profiles', profilesRouter);
 app.use('/api/auth', authRouter);
+app.use('/', itinRouter);
```

Comment:
the base route should be untouched, add something like '/itineraries' instead of just '/'